wrappers/mmapwrapper.h

```diff
@@ -174,6 +181,16 @@ namespace HL {
 	perror("MmapWrapper");
 	return nullptr;
       } else {
+#if HL_NAMED_HEAP
```

Comment:
As it stands, if `PR_SET_VMA` is undefined, AFAICT this will currently cause a compiler error - please add `defined(PR_SET_VMA)` as a condition.

Reply:
[Above](https://github.com/emeryberger/Heap-Layers/pull/52/files#diff-35bca1317c82b76e52fe08dd9696e4111dd111436609b6ce3e950b8cff690564R32) I define PR_SET_VMA/PR_SET_VMA_ANON_NAME constants.